fenix/app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -1046,6 +1046,18 @@ class DefaultSessionControlControllerTest {
         }
     }
 
+    @Test
+    fun handleNavigateSearch() {
+        createController().handleNavigateSearch()
+
+        verify {
+            navController.navigate(
+                match<NavDirections> { it.actionId == R.id.action_global_search_dialog },
+                any<NavOptions>(),
+            )
+        }
+    }
```

Comment:
should we add the metrics verification here too?

Reply:
Oh yes, I forgot about that.